TTHAnalysis/python/tools/leptonBuilderEWK.py

```diff
@@ -651,7 +651,6 @@ def _susyEWK_lepId_CBloose(lep):
             if not (lep.convVeto and lep.lostHits == 0): 
                 return False
             #if not lep.mvaIdSpring15 > -0.70+(-0.83+0.70)*(abs(lep.etaSc)>0.8)+(-0.92+0.83)*(abs(lep.etaSc)>1.479):
-            #    return False
```

Comment:
Can you please leave these commented lines for reference (as soon as we are close to final reading, at least)?